book/src/keymap.md

```diff
@@ -402,3 +404,35 @@ Keys to use within prompt, Remapping currently not supported.
 | `Enter`                                     | Open selected                                                           |
+
+# File explorer
+Keys to use within explorer, Remapping currently not supported.
+
+| Key                      | Description                                                            |
+| -----                    | -------------                                                          |
+| `Escape`                 | Back to editor                                                         |
+|  `Ctrl-c`                | Close explorer                                                         |
+|  `Enter`                 | Open file or toggle dir selected                                       |
+| `b`                      | Back to current root's parent                                          |
+| `f`                      | Filter items                                                           |
+| `z`                      | Fold currrent level                                                    |
+| `k`, `Shift-Tab`, `Up`   | select previous item                                                   |
+| `j`, `Tab`, `Down`       | select next item                                                       |
+| `h`                      | Scroll left                                                            |
+| `l`                      | Scroll right                                                           |
+| `G`                      | Move to last item                                                      |
+| `Ctrl-d`                 | Move down half page                                                    |
+| `Ctrl-u`                 | Move up half page                                                      |
+| `Shift-d`                | Move down a page                                                       |
+| `Shift-u`                | Move up a page                                                         |
+| `/`                      | Search item                                                            |
+| `?`                      | Search item reverse                                                    |
+| `n`                      | Repeat last search                                                     |
+| `Shift-n`                | Repeat last search reverse                                             |
+| `gg`                     | Move to first item                                                     |
+| `ge`                     | Move to last item                                                      |
+| `gc`                     | Make current dir as root dir                                           |
+| `mf`                     | Create new file under current item's parent                            |
+| `md`                     | Create new dir  under current item's parent                            |
+| `rf`                     | Remove file selected                                                   |
```

Comment:
a suggestion on usability, probably it would be good to include that the removal will prompt for confirmation
```md
| `rf`                     | Remove the selected file with confirmation
| `rd`                     | Remove the selected dir with confirmation
```
I mean, probably in the future there would be an option to not require confirmation, maybe some `rF` and/or `rD`